### PR TITLE
chip accessibility fix

### PR DESCRIPTION
**Pull request by tshimber**
Fixes https://github.com/telekom/scale/issues/2328
Chip keyboard handling fixed

### CODE REVIEW

**Review by amir-ba — COMMENTED** [review]
also, the prettier needs to be run for the branch to avoid pipeline fails

File: packages/components/src/components/chip/chip.tsx [hunk 1]
```diff
@@ -94,6 +94,19 @@ export class Chip {
...
+  };
+
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.code === 'Space') {
+      this.handleChangeEvent(event);
+    }
+  };
+
+  handleChangeEvent = (event: MouseEvent | KeyboardEvent): void => {
+    if (this.disabled && this.type === 'dynamic') {
```
[the remark this answers is not in the document]
Comment (tshimber) [reply packages/components/src/components/chip/chip.tsx:107]: these are different checks, but I wanted to discuss with you this type dynamic anyway. i don't get it's meaning so I don't really get checks either

**Comment by tshimber** [comment]
> also, the prettier needs to be run for the branch to avoid pipeline fails

no, it appeared to be the outdated snapshot. I've updated it, so you can retrigger pipeline, sorry. but it's also strange that i don't see command for it in package. i would also discuss tomorrow

File: packages/components/src/components/chip/chip.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (amir-ba) [reply packages/components/src/components/chip/chip.tsx:107]: I think it makes no logical sense we can discuss this.

File: packages/components/src/components/chip/chip.tsx [hunk 2]
```diff
@@ -94,6 +94,19 @@ export class Chip {
   };
 
   handleClick = (event: MouseEvent) => {
+    this.handleChangeEvent(event);
+  };
+
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.code === 'Space') {
```
[the remark this answers is not in the document]
Comment (amir-ba) [reply packages/components/src/components/chip/chip.tsx:101]: I checked with google-material-design guidelines. But we need to check this against ours  as well.

**Comment by amir-ba** [comment]
Dynamic Selection Standard version seems to stlil not be focusable

File: packages/components/src/components/chip/chip.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (tshimber) [reply packages/components/src/components/chip/chip.tsx:98]: @amir-ba what we've decided here? only space?..

**Comment by amir-ba** [comment]
@tshimber the ` uncommitted-changes ` is failing again .please address this.

**Comment by tshimber** [comment]
> @tshimber the `uncommitted-changes` is failing again .please address this.

my bad, forgot to run build here after prev fix. hopefully fixed now! could you run again please?

**Comment by amir-ba** [comment]
LGTM

**Pull request merged**